src/apps/service/apps/de/jobs.clj

```diff
@@ -15,22 +16,10 @@
             [apps.service.apps.de.jobs.io-tickets :as io-tickets]
             [apps.util.json :as json-util]))
 
-(defn- pre-process-jex-step
-  "Removes the input array of a fAPI step's config."
-  [{{step-type :type} :component :as step}]
-  (if (= step-type "fAPI")
-    (dissoc-in step [:config :input])
-    step))
-
-(defn- pre-process-jex-submission
-  "Finalizes the job for submission to the JEX."
-  [job]
-  (update-in job [:steps] (partial map pre-process-jex-step)))
-
```

Comment:
Removing support for fAPI jobs means that we no longer have to do any step preprocessing. 🎉